plugins/generator-1.16.5/forge-1.16.5/templates/armor.java.ftl

```diff
@@ -100,7 +100,7 @@ package ${package}.item;
 
 		<#if data.enableHelmet>
         elements.items.add(() ->
-			new ArmorItem(armormaterial, EquipmentSlotType.HEAD, new Item.Properties()<#if data.enableHelmet>.group(${data.creativeTab})</#if>) {

```

Comment:
Why did you delete this condition (the same question applies to other items)?

Reply:
Because on line 101 the condition is already here so there is no need to put it twice...

Reply:
(and same for other items, each time the condition is before the `elements.items.add()` line...